package.json

```diff
@@ -1,8 +1,35 @@
 {
+  "name": "web-gphoto2",
+  "version": "0.1.0",
+  "description": "WebAssembly implementation of gphoto2 and libusb to control DSLR cameras over USB on the Web",
+  "main": "ui/camera.js",
+  "types": "ui/libapi.mjs.d.ts",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/GoogleChromeLabs/web-gphoto2"
+  },
+  "keywords": [
+    "gphoto2",
+    "libusb",
+    "webassembly"
+  ],
+  "author": "Ingvar Stepanyan - RReverser - me@rreverser.com",
```

Comment:
Check out either JSON or shorthand npm syntax for people fields: https://docs.npmjs.com/cli/v9/configuring-npm/package-json#people-fields-author-contributors

I usually just put "Ingvar Stepanyan <me@rreverser.com>" which is valid shorthand.

Also add `contributors` field with yourself in the list too, after all, you're authoring the package :)